docs/standard/data/xml/how-to-perform-an-xslt-transformation-by-using-an-assembly.md

```diff
@@ -128,29 +128,29 @@ The XSLT compiler (xsltc.exe) compiles XSLT style sheets and generates an assemb
   
 1. Executing the following command from the command line creates two assemblies named `Transform.dll` and `Transform_Script1.dll` (This is the default behavior. Unless otherwise specified, the name of the class and the assembly defaults to the name of the main style sheet):  
   
-    ```  
+    ```console  
     xsltc /settings:script+ Transform.xsl  
-    ```  
+    ```
   
- The following command explicitly sets the class name to Transform:  
+    The following command explicitly sets the class name to Transform:  
   
-```  
-xsltc /settings:script+ /class:Transform Transform.xsl  
-```  
+    ```console  
+    xsltc /settings:script+ /class:Transform Transform.xsl  
+    ```  
   
 ### To include the compiled assembly as a reference when you compile your code.  
   
 1. You can include an assembly in Visual Studio by adding a reference in the Solution Explorer, or from the command line.  
   
 2. For the command line with C#, use the following:  
   
-    ```  
+    ```console  
     csc myCode.cs /r:system.dll;system.xml.dll;Transform.dll  
     ```  
   
 3. For the command line with Visual Basic, use the following  
   
-    ```  
+    ```console  
     vbc myCode.vb /r:system.dll;system.xml.dll;Transform.dll  
     ```  
   
```

Comment:
can you remove the numbered list from line 159?
the indentation is a bit off:
https://review.docs.microsoft.com/en-us/dotnet/standard/data/xml/how-to-perform-an-xslt-transformation-by-using-an-assembly?branch=pr-en-us-14482#to-use-the-compiled-assembly-in-your-code